adr/api/2020-03-03-API-Translations.md

```diff
@@ -0,0 +1,23 @@
+Date: 2020-03-03
+
+# API - Translations
+
+In power from version: 1.8.0 (?)
+
+Related PRs: #11136, #11144, #11157
+
+## Description
+
+### Problem to solve
+
+Provide unified way to manage translations for translatable entities in API. 
+
+### Possible solutions
+
+The translations from collection could be embedded as objects within a `ProductOption`/`ProductOptionValue` resource
```

Comment:
Should we consider placing an option for flatten a list of translations to the object itself (similar to https://medium.com/locastic/having-troubles-with-implementing-translations-in-apiplatform-7861294fb7c6)?

Reply:
We should consider it, but for sure not for the first version 😄